data/brands/amenity/university.json

```diff
@@ -43,6 +43,18 @@
         "name": "University of Phoenix",
         "short_name": "UOPX"
       }
+    },
+    {
+      "displayName": "University of Maryland, College Park",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "university",
+        "brand": "University of Maryland, College Park",
+        "brand:wikidata": "Q503415",
```

Comment:
A university is a chain brand is very special, and I just looked for the wikidata ```Q503415```, it refer to a single university of Maryland, but ```Q1480532``` can refer to the system of a series of Univ. Maryland.

>It is a pity that there is no such kind of chain university in my area, so I am afraid I can't understand why there is this type of data.
>And the University of California, Berkeley, Santa Barbara, these are obviously a series of appearances, and they are not included in the university. Therefore, I am very cautious about adding data to this file, amenity/university.json.